### ISSUE TITLE
MQTT playlist topic isn't always updating

### ISSUE THREAD

**Issue by Jchuchla** [FalconChristmas/fpp#1704]
**FPP Version and Hardware**: 
FPP 7.3.3 on Pi 3B+

**Describe the bug and Steps to reproduce**: 
MQTT playlist\name\status topic isn't always updating.  

If I trigger a sequence from remote falcon, the playlist\name\status topic changes to "Remote Falcon" as it should.  
When FPP switches back to the regularly scheduled playlist.  The playlist\name\status topic does not always update to show the active playlist.  

**Expected behavior**: 
The playlist\name\status topic should always report the currently active playlist regardless of how the playlist is triggered.

**Comment by ghormann** [comment]
I'm not an expert on remote falcon, but scanning the code it can do both `InsertImediate`  (Interrupt the current playlist) and `InsertAfterCurrent.`  Do you happen to know which is being used when you observe this behavior?

**Comment by Jchuchla** [comment]
I've never looked at the underlying code.  But I know there's an option to interrupt or queue it to play next.  I'm assuming that option corresponds with these methods in the code.  I used the play next option when I noticed the problem.
Just to be clear.  the MQTT correctly reported the switch to the RF playlist.  But it did not report correctly when it returned to the regularly scheduled playlist after the RF queue was done.
